README.md

```diff
@@ -80,12 +80,12 @@ To get started right away, jump to the [Quickstart](#quickstart) section. `buku`
 
 #### Dependencies
 
-| Feature | Dependency |
-| --- | --- |
-| Lang, SQLite | Python 3.7+ |
-| HTTPS | certifi, urllib3 |
-| Encryption | cryptography |
-| HTML | beautifulsoup4, html5lib |
+| Feature      | Dependency               |
```

Comment:
What's the problem here? It's renders perfectly. Please revert.

Reply:
It's auto formatted.
I'll revert it later.